docs/release_notes.rst

```diff
@@ -4,6 +4,13 @@ Release Notes
 **pydocstyle** version numbers follow the
 `Semantic Versioning <http://semver.org/>`_ specification.
 
+Current Development Version
+---------------------------
+
+Bug Fixes
+
+* Fix ``DeprecationWarning`` / ``SyntaxError`` "invalid escape sequence" with Python 3.6+ (#445).
```

Comment:
Please keep to 80 characters per line.

Reply:
Sure, fixed: 6fe013b7f4ad05a56837cb0576812e7212df3c52.